apps/web/pages/my-account/billing-details.vue

```diff
@@ -9,6 +16,7 @@
 
 <script setup lang="ts">
 import { AddressType } from '@plentymarkets/shop-api';
+const { t } = useI18n();
```

Comment:
use `$t` in the template and remove the `const { t } = useI18n();` we only `t` if its used in the script. if we use t in the script, we also use t without the $ in the template. Otherwise, vue imports it twice.